src/main/java/no/entur/uttu/export/netex/producer/line/JourneyPatternProducer.java

```diff
@@ -88,11 +99,30 @@ public org.rutebanken.netex.model.JourneyPattern produce(
     );
     context.notices.addAll(local.getNotices());
 
+    List<LinkInLinkSequence_VersionedChildStructure> linksInSequence;
+    if (osrmService.isEnabled()) {
+      linksInSequence = local
+              .getPointsInSequence()
+              .stream()
+              .map(spinjp -> mapServiceLinkInJourneyPattern(
+                      local.getRef(),
+                      spinjp,
+                      spinjp.getOrder() != local.getPointsInSequence().size() ? local
+                              .getPointsInSequence().get(spinjp.getOrder()) : null, context))
+              .filter(Objects::nonNull)
+              .collect(Collectors.toList());
+    } else {
+      linksInSequence = new ArrayList<>();
+    }
+    LinksInJourneyPattern_RelStructure linksInJourneyPattern_relStructure = new LinksInJourneyPattern_RelStructure()
+            .withServiceLinkInJourneyPatternOrTimingLinkInJourneyPattern(linksInSequence);
+
     return objectFactory
       .populate(new org.rutebanken.netex.model.JourneyPattern(), local)
       .withRouteRef(routeRef)
       .withName(objectFactory.createMultilingualString(local.getName()))
-      .withPointsInSequence(pointsInJourneyPattern_relStructure);
+      .withPointsInSequence(pointsInJourneyPattern_relStructure)
+      .withLinksInSequence(linksInSequence.isEmpty() ? null : linksInJourneyPattern_relStructure);
```

Comment:
This null thing was important, for some reason having linksInSequence with nothing inside would throw an exception